src/ape/api/providers.py

```diff
@@ -695,6 +726,20 @@ def get_code(self, address: str) -> bytes:
         return self.web3.eth.get_code(address)
 
     def get_storage_at(self, address: str, slot: int, **kwargs) -> bytes:
+        """
```

Comment:
@helloibis Ends up looking like this:

<img width="726" alt="Screen Shot 2022-08-11 at 08 46 07" src="https://user-images.githubusercontent.com/19540978/184148105-c4b3c2c6-aa38-4419-a143-17b4b67ce5e4.png">


Thoughts?

Reply:
@unparalleled-js yeah that's awesome :)